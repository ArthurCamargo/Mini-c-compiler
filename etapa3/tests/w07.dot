digraph G {
n0;
n0 -> n1;
n1;
n1 -> n2;
n2;
n2 [label="a"];
n1 [label="input"];
n0 [label="f"];

}