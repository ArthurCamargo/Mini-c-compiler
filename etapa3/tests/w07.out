0x5570860f6350
0x5570860f6350, 0x5570860f64a0
0x5570860f64a0
0x5570860f64a0, 0x5570860f6450
0x5570860f6450
0x5570860f6450 [label="a"];
0x5570860f64a0 [label="input"];
0x5570860f6350 [label="f"];
